examples/dotnet/rideshare/example/Program.cs

```diff
@@ -26,12 +30,83 @@ public static void Main(string[] args)
             scooterService.Order(2);
             return "Scooter ordered";
         });
-        
+
         app.MapGet("/car", () =>
         {
             carService.Order(3);
             return "Car ordered";
         });
+        
+        
+        app.MapGet("/pyroscope/cpu", (HttpRequest request) =>
+        {
+            var enable = request.Query["enable"] == "true";
+            Pyroscope.Profiler.Instance.SetCPUTrackingEnabled(enable);
+            return "OK";
+        });
```

Comment:
A nit: for the sake of demo we could also allow disabling profilers

Reply:
Sorry I don't understand the comment. if I pass something other then true to the query, it will disable profiling

Reply:
Oh right, my bad!